source/accessible-media/index.html.erb.md

```diff
@@ -19,55 +19,50 @@ contributors:
   - The Education and Outreach Working Group (<a href="https://www.w3.org/WAI/EO/">EOWG</a>)
 support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/">U.S. Access Board, WCAG TA Project</a>
 ---
-Captions and audio descriptions make online media accessible to people who have disabilities. This tutorial will demonstrate how to add captions and audio descriptions to online media.  Other accessibility techniques, such as transcripts and foreign-language subtitles, are included in the discussion.  Here are brief definitions of these media-accessibility approaches.
+Captions and audio descriptions make online media accessible to people with disabilities. This tutorial will demonstrate how to add captions and audio descriptions to online media. In addition to captions and audio descriptions, other techniques, such as transcripts and foreign-language subtitles are available to ensure more accessible media.
```

Comment:
- We use “people with disabilities” for consistency.
- I wanted to not use “discussion” in the sentence as this is advice.
- “Here are brief definitions of these media-accessibility approaches.” is superfluous.